### ISSUE TITLE
Release 1.0.14

### ISSUE THREAD

**Issue by jleach** [bcgov/bc-wallet-mobile#1679]
Test 1.0.13 Build 1400

- [ ] - [ ] Manual testings 
    - [ ] Showcase (Student/Lawyer)
    - [ ]  LSBC Test Environment
    - [ ]  LSBC Prod environment
    - [ ]  Accessibility: Voice Control, Labels
    - [ ]  Get Person Credential with BCSC App
- [ ] Automated testings are passing
- [ ] Mobile Verifier with Default and Dev Mode settings
- [ ] Connected experience (e.g.: messaging, proof request, wallet name/alias)
- [ ] Single Device experience using showcase (deep linking)
- [ ] Use names/values supporting Indigenous Languages

Change notes:
```text
- Accessibility improvements;
- Unlock Person credential with Digital Business Card
```

**Comment by jleach** [comment]
- [x] Manual testings 
    - [ ] Showcase (Student/Lawyer)
    - [ ] LSBC Test Environment
    - [x] LSBC Prod environment (tested by Kim, iOS and Android Ok)
    - [ ] Accessibility: Voice Control, Labels
    - [ ] Get Person Credential with BCSC App
    - [x] Get Digital Business Card
- [ ] Automated testings are passing
- [ ] Mobile Verifier with Default and Dev Mode settings
- [ ] Connected experience (e.g.: messaging, proof request, wallet name/alias)
- [ ] Single Device experience using showcase (deep linking)
- [ ] Use names/values supporting Indigenous Languages

**Comment by nodlesh** [comment]
- [x]  Manual testings 
    - [x] Showcase (Student/Lawyer) (Android and iOS, Android much slower than iOS) 
    - [ ]  LSBC Test Environment
    - [ ]  LSBC Prod environment
    - [ ]  Accessibility: Voice Control, Labels
    - [ ]  Get Person Credential with BCSC App
- [x] Automated testings are passing
- [ ] Mobile Verifier with Default and Dev Mode settings
- [ ] Connected experience (e.g.: messaging, proof request, wallet name/alias)
- [ ] Single Device experience using showcase (deep linking)
- [ ] Use names/values supporting Indigenous Languages

I will update this morning, but build 1400 didn't do well on Android in the automated regression tests. iOS was good and recent builds have been trending well for both platforms. I believe the Android issue maybe something to do the the test agents, but need to investigate further. I'm rerunning the job right now to see if it was an anomaly.

**Comment by bryce-mcmath** [comment]
- [x]  Manual testings 
    - [x] Showcase (Student/Lawyer) 
    - [ ]  LSBC Test Environment
    - [ ]  LSBC Prod environment
    - [x]  Accessibility: Voice Control, Labels
    - [x]  Get Person Credential with BCSC App
- [ ] Automated testings are passing
- [x] Mobile Verifier with Default and Dev Mode settings
- [x] Connected experience (e.g.: messaging, proof request, wallet name/alias)
- [ ] Single Device experience using showcase (deep linking) - the student showcase seems to be consistently failing with the deeplink flow. The credential offer is received, but it hangs while receiving the credential. Lawyer showcase works well so I don't think this is a bug in the wallet
- [x] Use names/values supporting Indigenous Languages

Tested on iOS and Android, both seem great to me, besides the student showcase deeplink issue, the only thing noteworthy for me is that the first initialization of the day is slow, all subsequent initializations are very fast. That makes sense to me with the changes Wade made though.

**Comment by nodlesh** [comment]
Update on the automated regression tests. The ngrok agent containers are failing to start because of an authentication issue. I don't have a fix for this yet, so the Android tests will continue to fail.  Based on the last few builds passing in the regression I would say build 1400 is the same.

**Comment by jleach** [comment]
Working with Peter in the background. He's not seeing the Person credential activate when he receives the business card.

**Comment by jleach** [comment]
Business Card issues fixed in 1.0.13 1403.

**Comment by jleach** [comment]
Held back 1.0.13 because we found an issue with the tutorial screens on iPad. Will skip 1.0.13 and release 1.0.14 with minimal testing as it only has superficial changes:

For the 1.0.14 BC Wallet release, we have these changes in BC Wallet:
```console
➜  bifold git:(main) git log --pretty=format:%s v1.0.13..v1.0.14
chore: specify android abi filters (#1693)
fix: update packages and adjust tour image sizing (#1692)
chore: bump build number (#1687)
feat: enable jest testing (#1689)
```

And these changes in Bifold
```console
➜  bifold git:(main) git log --pretty=format:%s v1.0.0-alpha.161..v1.0.0-alpha.163
fix: in app guides orientation on ipad (#1055)
fix:disabling going back function on connectionless proof flow (#1051)
```

My thought is all the only changes are the fix the iPad screen size issue and disabling the going back. I think if QA regression looks good and it works with the showcase we're good to release it.

**Comment by jleach** [comment]
Smoke test on 1.0.14 1418 Ok.

**Comment by nodlesh** [comment]
Automated regression tests on iOS are good. Android has been failing but not because of the app. I have a fix for the test agents coming today, so this weekends regression runs for Android should work. If we want to release today, then I'm confident that Android will pass as iOS is passing.

**Comment by jleach** [comment]
Released 1.0.14 1418
